### PR TITLE
Build - DS1001 - DocuSign R1 Release Items

### PR DESCRIPTION
**Pull request by sriharip-docusign**
Change description: Few bug fixes and features.

- Bug : [FLOW-270] - Phone Authentication is not working in certain International locales like Australia, France etc
- Bug : [FLOW-282] - Adding Tabs for a recipient has the same Data Labels
- Bug : [FLOW-285] - FLOW - data is replicating across multiple tabs
- Feature: [FLOW-275] - Restore v1 Create from Template Action
- Feature: [FLOW-245] - SMS authentication in Add Verification to recipient
- Feature: [FLOW-29] - Include all envelope statuses in Trigger dropdown

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/DocuSign/apiProperties.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
           "type": "oauthSetting",
           "oAuthSettings": {
             "identityProvider": "DocuSign",
-            "clientId": "bc0cb699-7d9f-4d83-80bc-f31c5ca820c7",
+            "clientId": "<<Please add your clientID here>>",
```
Comment (angelga-docusign) [inline certified-connectors/DocuSign/apiProperties.json:8]: Is this intentional?

Comment (sriharip-docusign) [reply certified-connectors/DocuSign/apiProperties.json:8]: Yes. There is no need to expose the client id in the config as it is configured using the ISV studio tool.

**Review by Amjed-Ayoub — APPROVED** [review]

File: certified-connectors/DocuSignDemo/apiDefinition.swagger.json [hunk 2]
```diff
@@ -2286,6 +2497,56 @@
...
         "x-ms-visibility": "internal"
       }
+    },
+    "/build_number": {
+      "get": {
+        "tags": [
+          "StaticResponse"
+        ],
+        "description": "Identifier for the currently deployed build.",
+        "summary": "Show build Number (For reference only. Not meant for including in a flow for execution)",
```
Comment (Amjed-Ayoub) [inline certified-connectors/DocuSignDemo/apiDefinition.swagger.json:2507]: @sriharip-docusign The 'summary' value goes over the character limit '80'. Please fix. thank you

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by sriharip-docusign** [comment]
@Amjed Ayoub (TEKsystems, Inc.) ***@***.***>
Just made a commit to shorten the summary to less than 80 chars.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @sriharip-docusign,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**